renderer/src/pages/Dashboard.tsx

```diff
@@ -0,0 +1,113 @@
+import { useEffect, useState, useCallback } from 'react'
+import {
+  getAllActivities, stopSaturnNode,
+  setFilAddress, getFilAddress,
+  getTotalEarnings, getTotalJobsCompleted
+} from '../lib/station-config'
+import { ActivityEventMessage } from '../typings'
+import ActivityLog from '../components/ActivityLog'
+import HeaderBackgroundImage from '../assets/img/header.png'
+import WalletIcon from '../assets/img/wallet.svg'
+import { useNavigate } from 'react-router-dom'
+
+const Dashboard = (): JSX.Element => {
+  const navigate = useNavigate()
+
+  const [address, setAddress] = useState<string | undefined>()
+  const [totalJobs, setTotalJobs] = useState<number>(0)
+  const [totalEarnings, setTotalEarnigs] = useState<number>(0)
+  const [activities, setActivities] = useState<ActivityEventMessage[]>([])
+  const shortAddress = (str: string | undefined) => str
+    ? str.substring(0, 4) + '...' + str.substring(str.length - 4, str.length)
+    : ''
+  const disconnect = async () => {
+    // TODO: move disconnect logic to backend
```

Comment:
The point here is if we want to trigger `stopSaturnNode` here. The alternative would be to expose an API responsible to stop Saturn (and other modules) and disconnect the wallet all at once, instead of making it explicitly in a frontend page. Thoughts??

Reply:
As long as there's no problem doing this in the frontend, I don't see why we should change it. We do own the frontend 100% so there's no risk (like with web services) where the backend must be in control of all the business logic